```
[2026-06-25T19:49:17.352Z] src/syn/test/elab_test.cc:14:10: error: use of private header from outside its module: 'flow/import.h' [-Wprivate-header]
[2026-06-25T19:49:17.352Z]    14 | #include "flow/import.h"
```
seems familiar from a PR from @hzeller

--- a/src/syn/BUILD
+++ b/src/syn/BUILD
@@ -23,7 +23,6 @@ cc_library(
         "src/flow/export.cc",
         "src/flow/export.h",
         "src/flow/import.cc",
-        "src/flow/import.h",
         "src/flow/liveness.cc",
         "src/flow/opt_gatefusion.cc",
         "src/flow/opt_gatefusion.h",
@@ -33,6 +32,7 @@ cc_library(
     hdrs = [
         "include/syn/synthesis.h",
         "src/flow/constant_fold.h",
+        "src/flow/import.h",
     ],
     includes = [
         "include",

